src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java

```diff
@@ -18,88 +19,113 @@
 public class ArticlePageTests extends NewTestTemplate {
 
   @BeforeMethod(alwaysRun = true)
-  public void optInMercury() {
+  public void prepareTest() {
     driver.manage().timeouts().pageLoadTimeout(100, TimeUnit.SECONDS);
-    BasePageObject.turnOnMercurySkin(driver, wikiURL);
   }
 
+  private boolean failTest = false;
+
   private static final String[]
       FOOTER_ELEMENTS =
       {"Games", "Movies", "TV", "Comics", "Music", "Books", "Lifestyle", "Full site", "Licensing",
        "Privacy Policy", "Feedback"};
 
   // APT01
-  @Test(groups = {"MercuryArticleTests_001", "MercuryArticleTests", "Mercury"})
-  public void MercuryArticleTests_001_LogoAndSearchButtonAreVisible() {
+  @Test(groups = {"MercuryArticleTest_001", "MercuryArticleTests", "Mercury"})
+  public void MercuryArticleTest_001_Logo_Search_TopContributors_FooterElements() {
```

Comment:
Use block of assertion instead of  if / else
Then you can remove the base.failTest(failTest); thing

Assertion.assertTrue(articlePage.isSearchButtonVisible())